packages/use-segment/src/__tests__/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { AnalyticsBrowser, Context } from '@segment/analytics-next'
 import { renderHook } from '@testing-library/react-hooks'
-import React, { ReactNode } from 'react'
+import { ReactNode } from 'react';
```

Comment:
intended ?

Reply:
fixed with the format https://github.com/scaleway/scaleway-lib/pull/696